js/wp-seo-metabox.js

```diff
@@ -250,33 +370,35 @@ function sanitize_title( title ) {
 	return title;
 }
 
-
+/**
+ * Updates the meta description in the snippet preview
+ */
 function yst_updateDesc() {
 	var desc = jQuery.trim(yst_clean(jQuery('#' + wpseoMetaboxL10n.field_prefix + 'metadesc').val()));
 	var divHtml = jQuery('<div />');
 	var snippet = jQuery('#wpseosnippet');
 
-	if (desc == '' && wpseoMetaboxL10n.wpseo_metadesc_template != '') {
+	if (desc === '' && wpseoMetaboxL10n.wpseo_metadesc_template !== '') {
 		desc = wpseoMetaboxL10n.wpseo_metadesc_template;
 	}
 
-	if (desc != '') {
+	if (desc !== '') {
 		desc = yst_replaceVariables(desc, function (desc) {
 			desc = divHtml.text(desc).html();
 			desc = yst_clean(desc);
 
 
-			var len = -1;
+			var len;
 			len = wpseoMetaboxL10n.wpseo_meta_desc_length - desc.length;
```

Comment:
I would change this into <code>var len = wpseoMetaboxL10n.wpseo_meta_desc_length - desc.length;</code> and remove line 391